.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: create release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Publish Docker Image to Github packages
+      uses: machine-learning-apps/gpr-docker-publish@master
```

Comment:
do we want to pin this?

Reply:
Maybe? I copied it from your release flow and changed the image name. :-P

Reply:
ah, we probably should for a number of obvious reasons, I'll update mine too.